### PR TITLE
Support disabling NLS message lookup

### PR DESCRIPTION
**Pull request by DanHeidinga**
Add a omrport_control option to disable NLS message catalog
parsing.  Disabling NLS message lookup is only effective if
it is done before the catalog is parsed.  Once parsed, this
option has no effect.

NLS messages are used for a variety of reasons but parsing
the catalog can be a measurable cost in fast startup scenarios.

This option allows the runtime to determine if NLS should
be disabled and the "default string" should be used instead.

Signed-off-by: Dan Heidinga <daniel_heidinga@ca.ibm.com>

### CODE REVIEW

**Comment by DanHeidinga** [comment]
This is base support to disable NLS message lookup.  There will also be a follow on PR in OpenJ9 to build on this to enable OpenJ9 to disable NLS

File: port/common/j9nls.c [hunk 1]
```diff
@@ -678,6 +678,11 @@ parse_catalog(struct OMRPortLibrary *portLibrary, uintptr_t flags, uint32_t modu
 	}
 	nls = &portLibrary->portGlobals->nls_data;
 
+	if (nls->isDisabled != 0) {
+		/* NLS is disabled - just return the default_string */
```
[the remark this answers is not in the document]
Comment (DanHeidinga) [reply port/common/j9nls.c]: Flipped the check against 0 as requested.

> Also does it make more sense to add an isEnabled and initialize it to 1 and have your option set it to 0?

I started with that idea as well but the port globals are memset to 0 so the code looked messier setting `isEnabled` to 1 in the NLS initialization code.  I'll change it if you want but I think this is clearer, especially given the limited number of places examining the `isDisabled` flag

[the remark this answers is not in the document]
Comment (charliegracie) [reply port/common/j9nls.c]: ok makes sense.  this change is ok

**Comment by DanHeidinga** [comment]
> ` ERROR: Error fetching remote repo 'origin'`

The two failed builds are due to issues fetching the git repos

**Comment by charliegracie** [comment]
@genie-omr build osx,plinux

**Pull request merged**